### PR TITLE
Fix subtitle translation

### PR DESCRIPTION
**Pull request by CleoAguiar**
- add solution

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

**Comment by lynxlynxlynx** [comment]
LGTM!

File: curriculum/challenges/portuguese/01-responsive-web-design/applied-accessibility/avoid-colorblindness-issues-by-using-sufficient-contrast.portuguese.md [hunk 1]
```diff
@@ -59,10 +59,10 @@ tests:
 
 </section>
 
-## Solution
+## Solução
 <section id='solution'>
 
 ```js
-// solution required
+var code = "body {color: hsl(0, 55%, 15%); background-color: hsl(120, 25%, 55%);}"
```
Comment (itaisteinherz) [inline curriculum/challenges/portuguese/01-responsive-web-design/applied-accessibility/avoid-colorblindness-issues-by-using-sufficient-contrast.portuguese.md]:
```suggestion
const code = `
	body {
		color: hsl(0, 55%, 15%);
		background-color: hsl(120, 25%, 55%);
	}
`;
```

This improves the code quality and readability a bit, in my opinion.

**Review by itaisteinherz — APPROVED** [review]
Looks great!

**Review by lynxlynxlynx — DISMISSED** [review]
Ups, upon a closer inspection there is a problem. The css in the solution may be fine, but the solution should be standalone — it should include whatever is in the challenge seed too.

**Comment by RandellDawson** [comment]
Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 🎉

**Pull request merged**